lai_s3/s3.py

```diff
@@ -1,41 +1,48 @@
 import botocore.exceptions
 import lightning as L
-from lightning.storage.payload import Payload
```

Comment:
Can we add tests to cover the image loading and dataframe loading code paths?

Reply:
so.

1. I couldn't find a public bucket to use so either we need to make one with the official lightning account and then use github actions with the proper access credentials or use one of our own accounts. Preferably the former.
2. We don't need to test dataframe loading as we don't provide an example for that and it would be an override for the __getitem__ method.